index.html

```diff
@@ -0,0 +1,40 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>ToDo List</title>
+    <link rel="stylesheet" href="index.css" />
+  </head>
+  <body>
+    <p id="demo">Input task and click on the button to populate the list</p>
+    <div class="searchLayout">
+      <div class="taskFieldWrapper">
+        <div class="taskField">
+          <label for="text">Enter your task</label>
+          <input type="text" name="text" id="text" placeholder="Enter Task" />
+        </div>
+        <button id="btn" class="searchButton" onclick="btnSubmit()">
+          Click Me
```

Comment:
- The id of the button is not used anywhere, so we can remove it.
- The text on the button is wrong.